src/components/QRCode/QRCode.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import qrImage from 'qr-image';
+import { connectTheme } from '../../utils';
+
+/*
+* Taken from:
+* https://github.com/jprichardson/react-qr
+* */
+
+class QRCode extends Component {
```

Comment:
This could be PureComponent right? Stateless component I think this would make it more performant